renku/cli/workflow.py

```diff
@@ -534,3 +555,41 @@ def export(workflow_name, format, output, values):
 
     if not output:
         click.echo(result.output)
+
+
+@workflow.command()
+@click.argument("paths", type=click.Path(exists=True, dir_okay=True), nargs=-1)
+@click.pass_context
+def inputs(ctx, paths):
+    """Show all inputs used by workflows.
+
+    <PATHS>    Limit results to these paths.
+    """
+    result = workflow_inputs_command().build().execute(paths=paths)
+
+    input_paths = result.output
+
+    click.echo("\n".join(input_paths))
+
+    ctx.exit(0 if not paths or len(input_paths) == len(paths) else 1)
```

Comment:
Do we need a similar check as the `output` to cover directories?

Reply:
This is how the original show inputs worked. I honestly don't know why it differs from output.